### ISSUE TITLE
[Bug]: Getting PasswordComplexityPolicy when upgrading past v2.49.5

### ISSUE THREAD

**Issue by draeron** [zitadel/zitadel#8402]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

Self-hosted

### Version

2.49.5

### Database

CockroachDB

### Database Version

23.2.9

### Describe the problem caused by this bug

Once I try to upgrade any version past `v2.49.5` I can't login into zitadel anymore, the website redirect to _Errors.PasswordComplexityPolicy.NotFound_.

```log
time="2024-08-08T04:22:48Z" level=error caller="/home/runner/work/zitadel/zitadel/internal/api/ui/login/renderer.go:345" error="ID=QUERY-63mtI Message=Errors.PasswordComplexityPolicy.NotFound Parent=(sql: no rows in result set)"

time="2024-08-08T04:22:48Z" level=warning msg="missing translation" args="map[]" caller="/home/runner/work/zitadel/zitadel/internal/i18n/translator.go:161" error="message \"Errors.PasswordComplexityPolicy.NotFound\" not found in language \"en\"" id=Errors.PasswordComplexityPolicy.NotFound

time="2024-08-08T04:22:51Z" level=error caller="/home/runner/work/zitadel/zitadel/internal/api/ui/login/renderer.go:345" error="ID=QUERY-63mtI Message=Errors.PasswordComplexityPolicy.NotFound Parent=(sql: no rows in result set)"

time="2024-08-08T04:22:51Z" level=warning msg="missing translation" args="map[]" caller="/home/runner/work/zitadel/zitadel/internal/i18n/translator.go:161" error="message \"Errors.PasswordComplexityPolicy.NotFound\" not found in language \"en\"" id=Errors.PasswordComplexityPolicy.NotFound
```

I have tried to modify the password policy while rollbacking into a previous version but that doesn't seems to help.

### To reproduce

- upgrade image to any version over 2.50.x
- restart
- try to login

### Screenshots

![image](https://github.com/user-attachments/assets/021a90fe-10c2-46de-a451-7f832044b4f5)

### Operating System

Docker on Alpine host

**Comment by stebenz** [comment]
Hi @draeron I found some error message which got mixed up, but this situation is still a little confusing.
First off, I created this PR [here](https://github.com/zitadel/zitadel/pull/8722) to correct the error messages.
I saw through that, that no the password complexity policy but the lockout policy is your problem, could you maybe try to create the lockout policy to resolve the immediate problem?
To get a better understanding of your situation, where you working on the database directly?

**Comment by draeron** [comment]
> To get a better understanding of your situation, where you working on the database directly?

No, just upgraded the docker image, I haven't touch directly to the database. I'm running as a single container with the `start` command. I did tried to start/setup with `--init-projections=true` but didn't solve my problem.

**Comment by stebenz** [comment]
@draeron Did you init and setup ZITADEL correctly as provided by our examples?
You could also use our docker compose example to see if you did something different https://zitadel.com/docs/self-hosting/deploy/compose
As it looks you have no LockoutPolicy, which could indicate that the setup which creates and prefills the projection is not existing.

**Comment by draeron** [comment]
After an upgrade to 2.5x or 2.6x, this what the Lockout Policy page looks like

![Image](https://github.com/user-attachments/assets/13c0fa2f-ea17-4488-ae20-8cb6e90fc14c)

Even though it's set in 2.49.5

![Image](https://github.com/user-attachments/assets/3ea8a525-b297-453e-ba18-e8a8f0d6fce2)

**Comment by stebenz** [comment]
Hi @draeron 
I will close this issue for now, does this still trouble you?
If so, please reopen the issue, so that we can take care of it.
